SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -39,20 +39,22 @@ There is a basic example in the _Examples_ section below. For more examples of h
 
 >**Note:** Your external SpatialOS components must have an ID between 1000 and 2000 to be registered by the pipeline.
 
-You set up your game to receive component updates through implementing callbacks for specified SpatialOS component IDs and operation types. The GDK detects your callbacks when it’s initialized and uses them for all received data relating to components with IDs in the range 1000 to 2000.
-To create callbacks, you need to create a class derived from the GDK `UOpCallbackTemplate` base class and implement `GetComponentId` and one or more of the callback methods. These callback methods are parameterized by the following network operation types:
+You set up your game to receive network operations using the `USpatialDispatcher::AddOpCallback` functions. These overloaded functions are parameterized with a `Worker_ComponentId` and a callback function const reference that takes one of the following network operation types as an argument:
 
-* `virtual void OnAddComponent(const Worker_AddComponentOp& Op) {}`
-* `virtual void OnRemoveComponent(const Worker_RemoveComponentOp& Op) {}`
-* `virtual void OnComponentUpdate(const Worker_ComponentUpdateOp& Op) {}`
-* `virtual void OnAuthorityChange(const Worker_AuthorityChangeOp& Op) {}`
-* `virtual void OnCommandRequest(const Worker_CommandRequestOp& Op) {}`
-* `virtual void OnCommandResponse(const Worker_CommandResponseOp& Op) {}`
+* `Worker_AddComponentOp`
+* `Worker_RemoveComponentOp`
+* `Worker_AuthorityChangeOp`
+* `Worker_ComponentUpdateOp`
+* `Worker_CommandRequestOp`
+* `Worker_CommandResponseOp`
 
+`Worker_ComponentId` and each network operation type are defined in the [Worker SDK in C’s API](https://docs.improbable.io/reference/latest/capi/api-reference).
 
-The `UOpCallbackTemplate` base class contains protected references to the `UWorld` and GDK `StaticComponentView` enabling you to write more meaningful callbacks.
+To ensure your callbacks will be triggered for initial network operations received by the SpatialOS worker connection, you will need to register the callbacks inside your game instance's `OnConnected` event callback.
```

Comment:
This is a super long sentence.
Suggest breaking it up and losing the predictive future (will) passive as it's a bit of a mind bend to read.
```suggestion
You need to ensure that the SpatialOS worker connection receives your callbacks initial network operations. To do this,  register the callbacks inside your game instance's `OnConnected` event callback.
```
Assuming I have understood this correctly. :-)

Reply:
More of a "you'll likely want to ensure", how to phrase that?

Reply:
If you want to ensure that the SpatialOS worker connection receives your callback for initial network operations, you need to register the callbacks inside your game instance's `OnConnected` event callback.

Reply:
Suggestion - it's one sentence (bad) but it's a bit easier to follow than the original, I think (good).